src/Config/Config.ts

```diff
@@ -209,6 +209,13 @@ export class BridgeConfigGitLab {
         this.instances = yaml.instances;
         this.webhook = yaml.webhook;
         this.userIdPrefix = yaml.userIdPrefix || "_gitlab_";
+
+        for (const name in this.instances) {
```

Comment:
This is fine, but we've generally moved to using `URL` for configuration these days.